packages/odyssey-react/src/components/Select/Select.module.scss

```diff
@@ -31,25 +88,25 @@
 
 .root {
   position: relative;
-  max-width: $max-line-length;
+  max-width: var(--FontMaxWidth);
   overflow: hidden;
-  font-size: $size-body-base;
+  font-size: var(--FontSize);
 
   &:focus,
   &.focused,
   .focus & {
-    @include border-radius($base-border-radius);
+    @include border-radius(var(--BorderRadius));
 
     outline: none;
-    box-shadow: 0 0 0 4px $color-primary-light;
+    box-shadow: 0 0 0 var(--FocusOutlineWidth) var(--ColorFocusPrimary);
 
     &.open {
-      box-shadow: 0 0 0 4px $color-primary-light,
-        0 2px 0 4px $color-primary-light;
+      box-shadow: 0 0 0 var(--FocusOutlineWidth) var(--ColorFocusPrimary),
```

Comment:
Maybe just `ColorFocus` here and then `ColorFocusInvalid` below? Not a blocker.

Reply:
@edburyenegren-okta thanks I'll take a look. This would be a topic to include with your proposal so we get clarity on when to specify further. If we always add something like `Base` to the suffix we're setup for success when another variant appears in the future for example. That comes at the cost of a more verbose variable name however.

Also consider that many of these variables **are not** related to focus, they are simply other UI effects and styles that happen to be using box-shadow and re-using the Focus tokens. This would be another issue to include in your proposal.

Reply:
Good call out. This is definitely a case of a value being used where a functional token doesn't exist. We can update this one after we have a better sense of how the tokens should cascade.